frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionDetails.tsx

```diff
@@ -58,26 +62,52 @@ function ModelActionDetails({
   actions,
   onEnableImplicitActions,
   onArchiveAction,
+  onDeleteAction,
 }: Props) {
+  const { show: askConfirmation, modalContent: confirmationModal } =
+    useConfirmation();
+
   const database = model.database();
   const hasActionsEnabled = database != null && database.hasActionsEnabled();
   const canWrite = model.canWriteActions();
-  const hasImplicitActions = actions.some(action => action.type === "implicit");
 
   const actionsSorted = useMemo(
     () => _.sortBy(actions, mostRecentFirst),
     [actions],
   );
 
+  const implicitActions = useMemo(
+    () => actions.filter(action => action.type === "implicit"),
+    [actions],
+  );
+
+  const onDeleteImplicitActions = useCallback(() => {
+    askConfirmation({
+      title: t`Disable basic actions`,
```

Comment:
IIRC, most confirmations have a question mark at the end of their title. This is from `<ConfirmContent />` which is used in quite a few places as well. You can see that the default value which is kind of a suggestion also includes a question mark.

https://github.com/metabase/metabase/blob/c91fc1dfac13407a657f2c472fe75a1c687d4678/frontend/src/metabase/components/ConfirmContent.tsx#L24

Other examples
- <img width="665" alt="image" src="https://user-images.githubusercontent.com/1937582/219767637-0f5e3b38-d4d2-450f-bc64-3c2e7df7ef83.png">
- <img width="674" alt="image" src="https://user-images.githubusercontent.com/1937582/219767726-ce9f16f2-a341-43f7-bc53-e449608fa39b.png">

CC @kdoh Not sure what's the current state of the the copy is, but these are patterns I've seen in our app.